### PR TITLE
selection & how to participate section

### PR DESCRIPTION
**Pull request by jules32**
Hi @stefaniebutland and @eeholmes , this is a starting draft we can post when we contact participants.

@stefaniebutland please add selection info directly to this branch following Weds Mentors Call.

Cheers,
Julie

### CODE REVIEW

**Comment by eeholmes** [comment]
@stefaniebutland is this ready to merge?

**Comment by stefaniebutland** [comment]
Not yet. Reviewing w Julie

**Comment by stefaniebutland** [comment]
to update
- Where: remotely, via Google Meet -> Zoom
- comment out Attend a “Why Openscapes” info session

File: champions.qmd [hunk 1]
```diff
@@ -145,3 +147,31 @@ The growing [NMFS Openscapes Mentor community](https://nmfs-openscapes.github.io
...
+
+### Decision criteria
+
+We selected the Cohorts with transparent decision criteria that were developed with the Mentors, Eli Holmes, and the Openscapes team.
+
+We prioritized participation by:
+
+-   People who committed to a Cohort date-time by marking "Yes" in the spreadsheet
+-   People who have not previously participated in an Openscapes Champions Cohort
+-   Supervisors - TODO Update
```
Comment (jules32) [inline champions.qmd:167]:
```suggestion
```

**Comment by jules32** [comment]
@stefaniebutland I had another look, please go ahead and merge when you're ready.

File: champions.qmd [hunk 2]
```diff
@@ -145,3 +147,30 @@ The growing [NMFS Openscapes Mentor community](https://nmfs-openscapes.github.io
...
+-   Strategic Initiative groups: Passive Acoustics (PAM), Active Acoustics (AA), Climate and Fisheries (FDD). Genomics ('omics).
+-   Office that has never participated or participated little: OPR, OST, NEFSC, AKRO, GARFO
+-   New hires
+-   Teams with NOAA staff
+-   Caps on count totals across Centers: High: no more than 18 participants per division; Low: if \< 5 signed up for one Center, accept all.
+
+Criteria that did NOT affect selection
+
+-   People who signed up as individuals or as part of a team were considered equally
+-   Date of signup
```
Comment (jules32) [inline champions.qmd:176]:
```suggestion
-   Date of signup

Waitlist decisions. If people declined, we selected people from the waitlist if: 

- same cohort, same division, not participated before, same team

```

**Pull request merged**